plugin/trino-delta-lake/src/test/resources/deltalake/clone_merge/clone_merge_deletion_vector_source/README.md

```diff
@@ -0,0 +1,18 @@
+Data generated using OSS Delta Lake 3.3.0:
+
+```sql
+CREATE TABLE clone_merge_deletion_vector_source (
+    id int, v string, part date)    
+    USING DELTA    
+    TBLPROPERTIES ('delta.enableDeletionVectors' = 'true');
+    
+INSERT INTO clone_merge_deletion_vector_source VALUES     
```

Comment:
This insert creates single rows in each parquet files, so following delete just removes entire files and there is no deletionVector in tansactionlog.
It should be something like:
```
{"add":{"path":"20250206_113723_00025_y7b6p_79b0624a-b032-4e9f-b67b-9c84b9960729","partitionValues":{},"size":511,"modificationTime":1738841844050,"dataChange":true,"stats":"{\"numRecords\":2,\"minValues\":{\"id\":2,\"v\":\"updated\",\"part\":\"2024-01-01\"},\"maxValues\":{\"id\":4,\"v\":\"updated\",\"part\":\"2024-02-02\"},\"nullCount\":{\"id\":0,\"v\":0,\"part\":0}}","tags":{},"deletionVector":{"storageType":"u","pathOrInlineDv":"-z*atcBlDyPB90fEl>c^","offset":1,"sizeInBytes":38,"cardinality":1}}}
```
Not sure how to enforce more values in one file thou. @ebyhr Do you know?

Reply:
Are you wanna see the cloned table read the source table DVs? 
It's not support to read the 'p' type of DV now which in the cloned table.
Even we support read the 'p' type DV, still, need to change the type to 'u' and change the path to relative path in DV, since we are loading the table from resource don't know the prefix of the absolute path. But that would need more extra logic to "allow read relative path as well" in the implementation of the support read absolute path DV.

Due to the constraints, seems now add the DV tests into product test is more natural to do, WDYT

Reply:
Filed #24946 , when this finalized we can add this test by modifying the reference path when loading tables.

Reply:
So cloned table will always have `p` type I assume? If that's the case then it's also ok that cloned table with vectors  would not be supported.  So either https://github.com/trinodb/trino/pull/24946 can be treated as prerequisite, or leave this test in current state add product test that shows Trino failure in such case.

Reply:
so far seems so. I prefer to update the test after `p` type dv read is supported

Reply:
Pls rebase on `master` to make use of https://github.com/trinodb/trino/pull/24946

Reply:
Updated, PTAL